fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -12,12 +12,14 @@ FIELDMAP_METHOD_OPT="FIELDMAP"
 SIEMENS_METHOD_OPT="SiemensFieldMap"
 GENERAL_ELECTRIC_METHOD_OPT="GeneralElectricFieldMap"
 SPIN_ECHO_METHOD_OPT="TOPUP"
+NONE_METHOD_OPT="NONE"
 
 ################################################ SUPPORT FUNCTIONS ##################################################
 
 # --------------------------------------------------------------------------------
 #  Load Function Libraries
 # --------------------------------------------------------------------------------
+
 source "${HCPPIPEDIR}/global/scripts/debug.shlib" "$@" # Debugging functions; also sources log.shlib
 
 Usage() {
```

Comment:
The parts in the Usage statement that refer to "readout distortion correction", should be "susceptibility distortion correction" instead.  (Currently, L's 49, 56, 59, and 63).  Looks like an incorrect copy/paste at some point in time from PreFreeSurferPipeline.sh.  Also, 'NONE' should probably we added as a listed mode.

Reply:
Added NONE. Should it say that it is only available for LegacyStyleData processing mode?

```
  echo "               \"${NONE_METHOD_OPT}\""
  echo "                 do not use any susceptibility distortion correction"
```

Reply:
Sure.

Reply:
Added.